File: factorial.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
	int n;
	cout<<"Enter a number: ";
	cin>>n;
	int fact=1;
	for(int i=n;i>=1;i--)
	{
		fact =*i;
	}
	
	cout<<"Factorial of "<<n<<" is "<<fact<<"\n";
	return 0;
}

File: Languages/Java/Factorial.java
import java.util.Scanner;
public class Factorial {
    int number;
    public void set_data(){
        Scanner sc = new Scanner(System.in);
        number=sc.nextInt();
        sc.close();
    }
    public void fact(){
        int fact =1;
        for(int i=2;i<=number;i++)
            fact=fact*i;
        System.out.println("Factorial is: "+fact);
    }
    public static void main(String[] args) {
        Factorial Fact = new Factorial();
        System.out.print("Enter a Number: ");
        Fact.set_data();
        Fact.fact();
    }
}
